pyproject.toml

```diff
@@ -76,6 +76,15 @@ docs = [
     "myst-parser==2.0.0",
     "IPython",
     "tensorflow==2.16.*",
+    "sphinx-tabs",
+]
+hpo = [
+    "bayesian-optimization==2.0.*",
+    "hyperopt==0.2.*",
+    "ConfigSpace==1.2.*",
+    "hpbandster==0.7.*",
+    "scipy==1.14.*",
+    "GPy==1.13.*"
 ]
```

Comment:
Did you set these manually or were they set by uv?

Reply:
Manually :) We talked about this before I think, I didn't change this specifically here, it shows up from my rebase onto ray-torchtrainer-integration